src/containers/ProjectDashboardContainer.jsx

```diff
@@ -21,7 +21,7 @@ const propTypes = {
 
 class ProjectDashboardContainer extends Component {
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
Mostly so that you aren't waiting too long between clicking a link and loading the new view. I think this is a pattern we use in everything else Zooniverse has built: start requesting data and setting state while the component is rendering. It's a small optimisation, but it avoids an annoying empty render if your API resource is already cached and available.